.vscode/launch.json

```diff
@@ -26,51 +26,21 @@
       "webRoot": "${workspaceFolder}/app/javascript/packs"
     },
     {
-      "name": "Debug Local File",
-      "type": "Ruby",
-      "request": "launch",
-      "cwd": "${workspaceRoot}",
-      "program": "${workspaceRoot}/main.rb"
-    },
-    {
-      "name": "Listen for rdebug-ide",
-      "type": "Ruby",
-      "request": "attach",
-      "cwd": "${workspaceRoot}",
-      "remoteHost": "127.0.0.1",
-      "remotePort": "1234",
-      "remoteWorkspaceRoot": "${workspaceRoot}"
-    },
-    {
-      "name": "Rails server",
```

Comment:
I omitted the "Rails server" task since 
- I couldn't get this to work with type `ruby_lsp` or `rdbg` 
- the app already has a standard command for running the app: `./bin/startup`.